clients/cli/src/register.rs

```diff
@@ -131,13 +133,21 @@ pub async fn register_node(
                 Ok(())
             }
             Err(e) => {
+                print_friendly_error();
                 eprintln!("Failed to register node: {}", e);
                 Err(e.into())
             }
         }
     }
 }
 
+fn print_friendly_error() {
+    // RGB: FF = 255, AA = 170, 00 = 0
+    println!("\x1b[38;2;255;170;0m{}\x1b[0m", LOGO_NAME);
+    println!("\x1b[38;2;255;170;0mWe'll be back shortly\x1b[0m");
+    println!("Our site is under unprecedented traffic. Thank you for your patience.\n");
```

Comment:
Change to “the Prover network’s orchestrater is under unprecedented traffic. Team has been notified. Thank you for your patience while issue is resolved”